core/cli/src/plugin/reduce-installations.ts

```diff
@@ -6,6 +6,11 @@ import { HookSchemas, HookOptions } from '@dotcom-tool-kit/schemas'
 import { Conflict, isConflict } from '@dotcom-tool-kit/conflict'
 import { groupBy } from 'lodash'
 
+// TODO:20240429:IM check if we should be passing this as an argument instead
+// of a global – are we ever reducing installations twice in one Tool Kit
+// invocation?
+const reducedInstallations: Set<string> = new Set()
```

Comment:
**suggestion**: i was thinking this would be in the `config` alongside the `resolutionTrackers` instead of being _global_ global

**nitpick**: `reducedInstallations` isn't quite an accurate name, it's not the installations we're tracking it's plugins that installations come from. maybe `reducedInstallationPlugins`?